### DISCUSSION TITLE
How to sync changes from md files into qmd

### DISCUSSION THREAD

**Discussion by abdelkareemkobo** [quarto-dev/quarto-cli#6265] [Q&A] [answered]
### Description

I am writing my blog with a note talking app that uses .md files only and want to add this notes into my blog that i build with quarto is there any way to my theses changes happen automatically without copy/paste!

**Comment by mcanouil** [comment]
Put the md file in your project and include the content using `{{< include notes.md >}}`, see <https://quarto.org/docs/authoring/includes.html>.

**Reply by abdelkareemkobo** [reply]
this will work as it copied my notes from the md file and put it into the .qmd, but i want to type my notes in the .md files without copy this file from my knowledge base which contains a lot of files in different folders and then copy it into the project of my blog and add this shortcut! this a lot of work is there any better solutions? and thanks for your help

**Reply by mcanouil** [reply]
Are you using Obsidian?

**Reply by abdelkareemkobo** [reply]
I am using logseq and obsidian and i tried to use qmd files with obsidian before and followed all the issues here and on obsidian forum but they are not working well

**Reply by mcanouil** [reply, marked as answer]
Quarto can use md files.
Now for Quarto to read them, it needs access to them.
You could write a pre-render script to take the md files and do whatever you want/need with them before Quarto does its job.

**Reply by abdelkareemkobo** [reply]
It works now, but there is another problem the images doesn't appear in .qmd files or .md 
![image](https://github.com/quarto-dev/quarto-cli/assets/72500236/d0161d59-229a-423f-9d92-5750da706049)
 
![image](https://github.com/quarto-dev/quarto-cli/assets/72500236/9b049e8d-398a-4087-8ff7-b231862b4f04)

**Comment by abdelkareemkobo** [comment]
> Put the md file in your project and include the content using `{{< include notes.md >}}`, see https://quarto.org/docs/authoring/includes.html.

should the notes.md file be in the same level of index.qmd!  is there any way to give it the path to the file ! if it in different level!

**Reply by cderv** [reply]
It is not yet well documented : You should be able to use `/` to start the path to consider it at project level. 

```markdown
{{< include /project/path/to/note.md >}}
```

Related to https://github.com/quarto-dev/quarto-cli/discussions/3784

This will be better documented. 
* https://github.com/quarto-dev/quarto-cli/issues/3788

**Reply by abdelkareemkobo** [reply]
I created a vault for obsidian that contains the my files and added quarto project inside it and want to add the path of my notes that is outside the project but in obsidian. in other way the note.md is outside the project folder !

**Reply by cderv** [reply]
> in other way the note.md is outside the project folder !

In that case I don't think that it is possible. Or you would need to use absolute path, but that is really not advisable. 

You should maybe consider pre render step to synchronize a note folder within your project with maybe symlink (or copy) of your note to be included. Just an idea.

**Reply by abdelkareemkobo** [reply]
OK i will handle this, the current problem is that the images of my blog post for the cover doesn't appear

**Reply by mcanouil** [reply]
Hard to say without actual code.

**Reply by abdelkareemkobo** [reply]
@mcanouil  here is the code https://github.com/abdelkareemkobo/Abdelkareem-NNs

**Reply by abdelkareemkobo** [reply]
> Hard to say without actual code.

i tried a lot of solutions but couldn't figure out what is wrong could you help me!

**Reply by mcanouil** [reply]
There are many things wrong in your repository:
- You are using/redefining at several level the same YAML options.
- You have typos in your options.
- You should not have `.Rdata` or `.Rhistory` in your project.

This being said, I don't see the issue of the featured image not showing up.
<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/3d42c1fe-da42-4444-973b-e066e3e6ab4d">

**Reply by abdelkareemkobo** [reply]
the problem with the fields options i wasn't selecting the images to be included in the field but fixed  it thanks for your help
